src/dynarec/rv64/dynarec_rv64_helper.c

```diff
@@ -190,12 +190,13 @@ static uintptr_t geted_32(dynarec_rv64_t* dyn, uintptr_t addr, int ninst, uint8_
                 int64_t tmp = F32S;
                 if (sib_reg!=4) {
                     if(tmp && ((tmp<-2048) || (tmp>maxval) || !i12)) {
-                        MOV32w(scratch, tmp);
+                        rv64_move32(dyn, ninst, scratch, tmp, 0);
```

Comment:
Why not using MOV32w macro anymore?

Reply:
ah ok. I guess a comment on that optim would be nice, for later when everyone will have forget this trick